egs2/dns_icassp21/enh1/README.md

```diff
@@ -0,0 +1,22 @@
+<!-- Generated by ./scripts/utils/show_enh_score.sh -->
+# RESULTS
+## Environments
+- date: `Thu Apr 21 21:49:46 UTC 2022`
+- python version: `3.7.4 (default, Aug 13 2019, 20:35:49)  [GCC 7.3.0]`
+- espnet version: `espnet 202204`
+- pytorch version: `pytorch 1.10.1+cu111`
+- Git hash: `a8e5d3d0ef346728a1be74166d4030370b7386cf`
+  - Commit date: `Sun Apr 24 04:35:33 2022 +0900`
+
+
+## enh_train_enh_tcn_tf_raw
+
+- config: ./conf/tuning/train_enh_tcn_tf.yaml
+- Pretrained model: https://huggingface.co/espnet/dns_icassp21_enh_train_enh_tcn_tf_raw
+
+
+|dataset|STOI|SAR|SDR|SIR|SI_SNR|
+|---|---|---|---|---|---|
+|enhanced_cv_synthetic|0.93|18.96|18.96|0.00|18.79|
+|enhanced_tt_synthetic_track_1|0.77|14.19|14.19|0.00|12.15|
```

Comment:
```suggestion
|dataset|STOI|SAR|SDR|SI_SNR|
|---|---|---|---|---|
|enhanced_cv_synthetic|0.93|18.96|18.96|18.79|
|enhanced_tt_synthetic_track_1|0.77|14.19|14.19|2.15|
```

Reply:
I think you can also remove the SAR column, as it is regarded as the same as  SDR in single-source cases.